forms-flow-web/src/styles.scss

```diff
@@ -32,10 +33,13 @@ $danger: $red;
 $light: $gray-100;
 $dark: $gray-800;
 
-@import "~bootstrap/scss/bootstrap.scss";
-@import "~font-awesome/scss/font-awesome";
-@import "~formiojs/dist/formio.full.min.css";
-
+// @import "~bootstrap/scss/bootstrap.scss";
+// @import "~font-awesome/css/font-awesome.css";
+// @import "~formiojs/dist/formio.full.min.css";
+@import url(https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css);
```

Comment:
Just a suggestion, we should maximum avoid CSS @import, it increases the loading time
ref:
1.https://www.giftofspeed.com/avoid-using-css-import/
2.https://oxosolutions.com/avoid-css-import/

Reply:
Sure will check and update. :)